edx_exams/apps/lti/views.py

```diff
@@ -303,6 +298,11 @@ def end_assessment(request, attempt_id):
     # Platform MUST send an End Assessment message to the Proctoring Tool. Otherwise, the Assessment Platform can
     # complete its normal post-assessment flow.
     if end_assessment_return:
+        # user is authenticated via JWT so use that to create a
```

Comment:
I moved this since it's only necessary if we're performing an end assessment launch